styx-api-service/src/test/java/com/spotify/styx/api/workflow/WorkflowInitializerTest.java

```diff
@@ -75,20 +74,6 @@ public void setUp() throws Exception {
         a.<TransactionFunction>getArgument(0).apply(transaction));
   }
 
-  @Test
-  public void shouldStoreNewWorkflowAndUpdateNextNaturalTrigger() throws IOException, WorkflowInitializationException {
```

Comment:
Or update it to match current behavior?

Reply:
You already added one when working on the PR.

Reply:
Ok